vpr/src/base/read_options.cpp

```diff
@@ -1897,6 +1932,28 @@ argparse::ArgumentParser create_arg_parser(std::string prog_name, t_options& arg
         .default_value("off")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    route_grp.add_argument<e_node_reorder_algorithm, ParseNodeReorderAlgorithm>(args.reorder_rr_graph_nodes_algorithm, "--reorder_rr_graph_nodes_algorithm")
+        .help(
+            "Specifies the node reordering algorithm to use.\n"
+            " * none: don't reorder nodes\n"
+            " * degree_bfs: sort by degree and then by BFS\n"
+            " * random_shuffle: a random shuffle\n")
+        .default_value("none")
+        .choices({"none", "degree_bfs", "random_shuffle"})
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    route_grp.add_argument(args.reorder_rr_graph_nodes_threshold, "--reorder_rr_graph_nodes_threshold")
+        .help(
+            "Reorder rr_graph nodes to optimize memory layout above this number of nodes.")
+        .default_value("0")
+        .show_in(argparse::ShowIn::HELP_ONLY);
+
+    route_grp.add_argument(args.reorder_rr_graph_nodes_seed, "--reorder_rr_graph_nodes_seed")
```

Comment:
Should comment why the rr_graph reordering algorithm needs its own seed.  Is it so you can get a deterministically re-ordered routing graph and hence load in a prior routing generated with the same reordering algorithm?  If there isn't a strong reason for the algorithm to have its own seed I'd just use the regular random number generator (and implicitly, its seed).

Reply:
The `--seed` flag help says "Placement random number generator seed" and is in the "placement options" group. I considered making that just a general seed, but I can imagine wanting to try the same reordering with different SA seeds, and vice versa, so I just added another argument.

If this isn't a good reason, I can use the other flag and change the documentation and grouping.

Reply:
I'm OK with leaving it as is then.  Didn't realize we called seed just placement seed; I don't have a really strong opinion about one seed vs. several so if you like this it's fine with me.